llama_hub/tools/waii/base.py

```diff
@@ -34,6 +34,32 @@ def __init__(
         WAII.Database.activate_connection(key=database_key)
         self.verbose = verbose
 
+    def _try_display(self, obj):
+        try:
+            # display df if it is available
+            if self.verbose:
+                display(obj)
+        except:
```

Comment:
why swallow the try/except here? is there an if/else statement instead?

Reply:
The reason is I only want to display the data frame with verbose=True. And if the tool logic runs outside of IPython, there's no `display` function. 

Alternatively, I can check if the function `display` exists or not, maybe that causes less confusion. I can also check if it is inside an IPython environment, but it seems more complex. e.g. https://discourse.jupyter.org/t/find-out-if-my-code-runs-inside-a-notebook-or-jupyter-lab/6935/7 

Let me know if you have any other ideas for this.

Reply:
ah i see. ok no problem